server/validation/validation.ts

```diff
@@ -61,15 +61,18 @@ export function validateAsync<T>(
   )
   Validator.registerAsyncImplicit(
     'single-main-building',
-    async (subCode, courtId, request, passes) => {
+    async (subCode, requirement, request, passes) => {
+      const [courtId, allowedBuildingId]: [string, string] = requirement.split(',') as [string, string]
```

Comment:
`requirement` is a string as defined by the validatorjs lib, and we need to pass both the current court id (to perform the lookup) and also the current building id (to ignore the current building) I've passed them both in the string as comma separated values.  I really don't like it but couldn't think of a better way.